opentreemap/treemap/decorators.py

```diff
@@ -168,12 +168,9 @@ def run_and_catch_validations(*args, **kwargs):
             return req(*args, **kwargs)
         except ValidationError as e:
             if hasattr(e, 'message_dict'):
+                message_dict['fieldErrors'] = e.message_dict
                 message_dict['globalErrors'] = [_(
                     'One or more of the specified values are invalid.')]
-                if 'globalErrors' in e.message_dict:
-                    message_dict['globalErrors'] += \
-                        e.message_dict.pop('globalErrors')
-                message_dict['fieldErrors'] = e.message_dict
```

Comment:
This may be worth keeping: "Change the 404 decorator to receive additional global errors
  along with field errors" from https://github.com/jwalgran/otm-core/commit/ab457ea78c45f41b226d5e4bc85032aea271cdba